Model/Page.php

```diff
@@ -134,6 +134,11 @@
     protected $target;
 
     /**
+     * @var int
+     */
+    protected $targetRedirectType = 302;
```

Comment:
My reasoning:
- http://fabien.potencier.org/pragmatism-over-theory-protected-vs-private.html
- close the API over having it consistent.